src/core/reactor_backend.cc

```diff
@@ -550,6 +557,10 @@ class aio_pollable_fd_state : public pollable_fd_state {
     future<> get_completion_future(int events) {
         return get_desc(events)->get_future();
     }
+    void abort() {
```

Comment:
ditto - optional exception param, and `noexcept`

Reply:
Exceptional param and noexcept flag added to `aio_pollable_fd_state::abort` and `epoll_pollable_fd_state::abort`